steps.ps1

```diff
@@ -13,24 +13,35 @@ if (($null -eq $PROJECT_NAME) -or ("" -eq $PROJECT_NAME))
   throw "Project Name not provided"
 }
 
-$headers = @{
-  'x-api-key'    = $env:LAZY_API_KEY
-  'Content-Type' = 'application/json'
+# Check if project.ps1 exists
+$projectScriptPath = $env:ACTION_PATH + "/project.ps1"
+Write-Output "Checking if project script exists at path: $projectScriptPath"
+if (-Not (Test-Path $projectScriptPath)) {
+    throw "Error: Script project.ps1 not found at path: $projectScriptPath. Ensure the script is available."
 }
 
-$Body = `
-@{"SpaceName"   = "$SPACE_NAME";
-  "ProjectName" = "$PROJECT_NAME";
-}
-
-Write-Output "Octopus Project Request Body"
-Write-Output $Body
+# Preparing to call project.ps1 with parameters
+Write-Output "=== Preparing to Call Project Script ==="
+Write-Output "Calling project.ps1 with parameters:"
+Write-Output "  SpaceName: $SPACE_NAME"
+Write-Output "  ProjectName: $PROJECT_NAME"
 
-$octoProjectEndpoint = "https://$env:LAZY_API_URL/octopus/project?user=$env:GITHUB_ACTOR"
+# Define parameters to pass to project.ps1
+$scriptParams = @{
+    "SpaceName"   = $SPACE_NAME
+    "ProjectName" = $PROJECT_NAME
+}
 
-Write-Output "Lazy API URL $octoProjectEndpoint"
+# Call the project.ps1 script and capture the output
+Write-Output "Executing project.ps1..."
+$projectOutput = & $projectScriptPath @scriptParams
```

Comment:
use ine or iee https://github.com/mklement0/Native.

You don't need lines 39-42 if you use that.